source/xep/omemo/lurch/lurch.c

```diff
@@ -1759,12 +1756,42 @@ static void lurch_message_decrypt(PurpleConnection * gc_p, xmlnode ** msg_stanza
       purple_conv_present_error(sender, purple_connection_get_account(gc_p), "Received encrypted message in blacklisted conversation.");
     }
   }
-#if 0
   else if (!g_strcmp0(type, "groupchat")) {
+
     split = g_strsplit(from, "/", 2);
     room_name = split[0];
     buddy_nick = split[1];
 
+    /*
+    addresses_node_p = xmlnode_get_child(*msg_stanza_pp, "addresses");
+    if(!addresses_node_p)
+    {
+      err_msg_dbg = g_strdup_printf("could not find addresses tag");
+      goto cleanup;
+    }
+
+    address_node_p = xmlnode_get_child(addresses_node_p, "address");
+    if(!address_node_p)
+    {
+      err_msg_dbg = g_strdup_printf("could not find address tag");
+      goto cleanup;
+    }
+
+    addr_type = xmlnode_get_attrib(address_node_p, "type");
+    if(!addr_type)
+    {
+      err_msg_dbg = g_strdup_printf("could not find type attrib type in address tag");
+      goto cleanup;
+    }
+
+    addr_jid = xmlnode_get_attrib(address_node_p, "jid");
+    if(!addr_jid)
+    {
+      err_msg_dbg = g_strdup_printf("could not find type attrib jid in address tag");
+      goto cleanup;
+    }
+    */
+
```

Comment:
Is this code from the original lurch.c file? The goal should be to have zero changes in the future in that file, when also group omemo handling is implemented to use lurch as an git-submodule, instead of a copy in the Shmoose repo.

Reply:
This code is from original lurch file. Group chat was commented in current Schmoose implementation. I just reduced  the commented part.